README.md

```diff
@@ -1,4 +1,4 @@
-Jasmine-Node Helpers
+Jasmine Test Helpers
```

Comment:
No, we should rename the project.  This applies to more than just `jasmine-node`.